flexmeasures/ui/utils/color_defaults.py

```diff
@@ -1,13 +1,14 @@
 from flexmeasures.data.models.user import Account
+from typing import Optional
 
 
-def get_color_settings(account: Account | None) -> dict:
+def get_color_settings(account: Optional[Account]) -> dict:
```

Comment:
I believe this should be resolved by `from __future__ import annotations`, and then we can keep the new `| None` notation

Reply:
That makes sense, i can implement that